docs/elements.md

```diff
@@ -1652,20 +1651,47 @@ def generate(data):
     data["params"]["matrix"] = pl.to_json(mat)
 ```
 
+---
+
+**question.html**
+
+```html
+<pl-graph params-name="random-graph"></pl-graph>
```

Comment:
This needs a `params-type` attribute, correct?

Reply:
Ah whoops, good catch

Reply:
Fixed!